src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -46,6 +45,8 @@ public function convert(SpanDataInterface $span)
         $parentSpanId = $span->getParentSpanId();
         $spanId = $span->getContext()->getSpanID();
 
+        //TODO - determine if any of the below commented out code is still needed
+
         // foreach ($span->getEvents() as $event) {
```

Comment:
This is the commented out code that was here before. Not sure what should be done with it